src/Perlang.Tests.Integration/ReservedKeywords/ReservedKeywordsTests.cs

```diff
@@ -485,7 +485,7 @@ public void function_return_type_detects_reserved_keyword_char()
         {
             string source = @"
                 fun foo(): char {
-                    return 123.45;
+                    return 123;
```

Comment:
These are just simplifications to avoid making these tests use constructs which can have locale-dependent bugs. (the kind of things fixed in #261)